cluster/cluster.yaml

```diff
@@ -2580,22 +3335,37 @@ Resources:
             Action:
             - "kms:Decrypt"
 Outputs:
-  MasterIAMRole:
+{{- if eq .Cluster.Provider "zalando-eks" }}
+  EKSControlPlaneEndpoint:
     Export:
-      Name: '{{.Cluster.ID}}:master-iam-role'
-    Value: !Ref MasterIAMRole
-  WorkerIAMRole:
+      Name: "{{.Cluster.ID}}:eks-control-plane-endpoint"
+    Value: !GetAtt EKSCluster.Endpoint
+  # EKSServiceIpv6Cidr:
+  #   Export:
+  #     Name: "{{.Cluster.ID}}:eks-service-cidr"
+  #   Value: !GetAtt EKSCluster.KubernetesNetworkConfig.ServiceIpv6Cidr
+  EKSOpenIdConnectIssuerUrl:
     Export:
-      Name: '{{.Cluster.ID}}:worker-iam-role'
-    Value: !Ref WorkerIAMRole
-  RemoteFilesEncryptionKey:
+      Name: "{{.Cluster.ID}}:eks-oidc-url"
```

Comment:
Should we use `.Cluster.Name` instead of `.ID` here (and other places?). PR that introduced the field: https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/829/files#diff-61dbe200b05c4fa62230dc5e9efedf4ac8d35088d3dfb7a1cde9163b79b41066